test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -337,7 +338,7 @@ func checkLifecycleHookResponses(ctx context.Context, c client.Client, namespace
 	for hookName, expectedResponse := range expectedHookResponses {
 		actualResponse, ok := responseData[hookName+"-actualResponseStatus"]
 		if !ok {
-			return errors.Errorf("hook %s call not recorded in configMap %s/%s", hookName, namespace, clusterName+"-hookresponses")
+			return errors.Errorf("hook %s call not recorded in configMap %s", hookName, klog.KRef(namespace, clusterName+"-hookresponses").String())
```

Comment:
```suggestion
			return errors.Errorf("hook %s call not recorded in configMap %s", hookName, klog.KRef(namespace, clusterName+"-hookresponses"))
```

As it implements the `Stringer ` interface, we should not need to add `.String()` everywhere?

It would be also okay to keep it for being more implicit 👍

Reply:
+1 to drop it

Reply:
done